docs/design.rst

```diff
@@ -50,21 +50,21 @@ Kafka Coordinator Master Election
 Kafka based master election is chosen when ``<kafkastore.connection.url>`` is not configured and
 has the Kafka bootstrap brokers ``<kafkastore.bootstrap.servers>`` specified. The kafka group
 protocol, chooses one amongst the master eligible nodes ``master.eligibility=true`` as the master. Kafka-based master
-election can be used in cases where ZooKeeper is not available, for example for hosted or cloud
-Kafka environments, or if access to ZooKeeper has been locked down.
+election can be used in cases where |zk| is not available, for example for hosted or cloud
+Kafka environments, or if access to |zk| has been locked down.
 
-Zookeeper Master Election
+|zk| Master Election
 +++++++++++++++++++++++++
 
 .. figure:: schema-registry-design.png
    :align: center
 
-   Zookeeper based Schema Registry
+   ZooKeeper based Schema Registry
```

Comment:
`|sr|`?

Reply:
yeah they don't work in examples or captions at this point